### PR TITLE
fix(core): Handle websocket connection error more gracefully in task runners

### PR DESCRIPTION
**Pull request by ivov**
https://linear.app/n8n/issue/CAT-318

### CODE REVIEW

File: packages/@n8n/task-runner/src/task-runner.ts [hunk 1]
```diff
@@ -88,6 +88,18 @@ export abstract class TaskRunner {
 			},
 			maxPayload: opts.maxPayloadSize,
 		});
+
+		this.ws.addEventListener('error', (error) => {
+			if (error.message?.includes('ECONNREFUSED') || error.message?.includes('ENOTFOUND')) {
```
Comment (tomi) [inline packages/@n8n/task-runner/src/task-runner.ts:93]: We could check `error.code` instead of using `message`

Comment (ivov) [reply packages/@n8n/task-runner/src/task-runner.ts]: [85e9c72](https://github.com/n8n-io/n8n/pull/11635/commits/85e9c72acab3d183eca90385db36222e9df78172)

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.68.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.68.0)

**Pull request merged**